docs/_markbind/layouts/devGuide/header.md

```diff
@@ -0,0 +1,19 @@
+<header>
+  <link rel="stylesheet" href="{{baseUrl}}/css/main.css">
+  <navbar type="dark">
+    <a slot="brand" href="{{baseUrl}}/index.html" title="Home" class="navbar-brand"><img src="{{baseUrl}}/images/logo-darkbackground.png" height="20" /></a>
```

Comment:
Minor nit: remove the backlash in img to be in line with the other img tags

Reply:
https://www.w3.org/TR/html5/syntax.html#writing-html-documents-elements

The / character is redundant, and the rest of the code doesn't use / in img tags as well as well.